mappings/base_mapping.py

```diff
@@ -29,6 +29,22 @@ def applyMapping(self):
 
     def applyFormatting(self):
         return self.record
+    
+    def format_identifiers(self, identifiers, source):
+        identifier_type_dict = {8: 'issn', 10: 'isbn', 13: 'isbn'}
+        for index, identifier in enumerate(identifiers):
+            count_digits = 0
+            if '/' not in identifier:
+                for char in identifier:
+                    if char.isdigit():
+                        count_digits += 1
```

Comment:
hmm could there be cases where an identifier has 8 digits but also has other alpha non-numeric characters?

we could simplify this and use list comprehension: 

`numeric_identifier = [digit for char in identifier if char.isdigit()]`

you can then use `len` to check the number of digits. 

I also recommend using the get method with dictionaries, which can be helpful when we need to default to a specific value. We can do things like:

`identifier_type = identifier_type_dict.get(len(numeric_identifier), source)`

Reply:
This sounds like a good idea to simplify and clean up this method.